### DISCUSSION TITLE
ensure plugin issue

### DISCUSSION THREAD

**Discussion by zoe-jobson-IL** [artilleryio/artillery#3340] [Artillery]
Hi I have been investigating adding the ensure plug in into some of our tests, one run with it in is here https://app.artillery.io/share/sh_08fc52a511d5c60f91078a0c44f1d281556af003b046e7db2b32e9db63442a38  - the trouble is that our items being checked have [] in the name and I think  its throwing off the code. If there a list of safe characters we can use ? or can you fix the issue.

**Comment by hassy** [comment]
hi @zoe-jobson-IL 👋 it should work even with those characters, you'll just need to make sure to include the full metric name in the expression, e.g.:

```
browser.step.[App sec labs] User waits for lab to be loaded (userWaitsForLabToBeLoaded) < 10000
```

should have `max` at the end of the metric name if you want to check `max` of that metric:

```
browser.step.[App sec labs] User waits for lab to be loaded (userWaitsForLabToBeLoaded).max < 10000
```

You can use `p99` or `p95` etc instead of `max` there as well.
